layouts/partials/page/header.html

```diff
@@ -1,8 +1,19 @@
-<h1 class="page-title offendingHeading">
 {{ if eq (.Scratch.Get "scope") "single" }}
-    {{ .Parent.Title }}
+    <h1 class="page-title offendingHeading">
+        {{ .Parent.Title }}
+    </h1>
 {{ else }}
-    {{ .Title}}
+    {{ $roles := .Params.roles | default "All Roles" }}
+    {{ $pageLink :=  .Page.RelPermalink }}
+    {{ $articleId := .Params.id | default .File.Path }}
+    {{ $slug := (partial "common/slug" .) }}
+
+    <div class="page title" data-roles="{{ $roles }}" data-link="{{ $pageLink }}" permalink="{{ .Page.RelPermalink }}" data-disable-article-bar="true">
```

Comment:
Need to wrap the title with the least amount of data to render the edit icon. When we render the WYSIWYG editor, it will not use the data in this div.

Not part of ticket, but this will allow us to add a permalink copy in the future.